app/controllers/api/v1/application_controller.rb

```diff
@@ -25,9 +25,13 @@ def authentication_error
       render json: { error: I18n.t('errors.401') }, status: 401
     end
 
-    def record_not_found_error
+    def existence_error
       render json: { error: I18n.t('errors.404') }, status: 404
     end
 
+    def conflict_error
+      render json: { error: I18n.t('errors.409') }, status: 409
+    end
```

Comment:
I think I'll leave it there for now since that's where the rest of the HTTPS error responses are.